src/js/fotorama.js

```diff
@@ -423,7 +423,34 @@ jQuery.Fotorama = function ($fotorama, opts) {
     }
   }
 
+  /**
+   * Defines whether fotorama in visible viewport.
+   * @param el - DOM element or jQuery object.
+   * @returns {boolean}
+   */
+  function isInViewport() {
```

Comment:
all objects, variables used in your function must be in parameters. it rather useful because not affect on global scope and i think it more transparent code